src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo"
+        >
+          <img
+            src="./images/logo.png"
+            alt="Logo"
+            class="header__image"
+          />
+        </a>
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item is-active">
```

Comment:
Fix classes everywhere
```suggestion
            <li class="nav__item">
```